src/lib/date.ts

```diff
@@ -3,25 +3,22 @@ import moment from "moment"
 export function exhibitionPeriod(startAt, endAt) {
   const startMoment = moment(startAt)
   const endMoment = moment(endAt)
-  const thisMoment = moment.utc(Date.now())
+  const thisMoment = moment()
```

Comment:
Is there a reason we don't want to use the UTC date?

Reply:
it doesn't do anything. Moment is smart enough to compare `moment()` to any date provided because `moment()` represents "now" whichever timezone is provided.

Reply:
Neat!

Reply:
Although looking at the failing test, maybe I'm wrong!